clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -37,6 +37,7 @@
   ],
   "icon": "resources/logo.png",
   "main": "./dist/extension.js",
+  "browser": "./dist/web/extension.js",
```

Comment:
How is this bundled in the CI?

Reply:
We'll need an extra step in `.github/workflows/build.yml` that will produce a new vsix package, packaged with `--target web` option.

Reply:
Let's do that. We will need it for publishing the WEB vsix.

Reply:
Added the step to run the package for web.